test/bats/gql/transactions.gql

```diff
@@ -0,0 +1,63 @@
+query transactions($walletIds: [WalletId], $first: Int, $after: String) {
```

Comment:
we generally use a pattern to have a single `$input` as argument for queries. I think it's good to use that consistently across our different apps